recommenders/datasets/movielens.py

```diff
@@ -100,6 +96,11 @@ def item_has_header(self):
     "20m": _DataFormat(",", "ml-20m/ratings.csv", True, ",", "ml-20m/movies.csv", True),
 }
 
+# Fake data for testing only
+MOCK_DATA_FORMAT = {
```

Comment:
why not just add this to data_format?

Reply:
The composition of the object is quite different: `DATA_FORMAT` uses an instance of `_DataFormat` object which contains data deserialization controls that doesn't fit the need to create a mock data, where you just need to provide the data size and seed for the pRNG. I figured it's simpler to separate the two to make the client of these objects easier.

Reply:
fair, i'm just lazy i guess =)